src/css/controls.less

```diff
@@ -0,0 +1,45 @@
+@import "shared-imports/mixins";
+
+// TO REMOVE
+@import "controls/imports/jwplayerelement";
```

Comment:
> TO REMOVE?
Yes please. "jwplayerelement" makes no sense - most of the classes styles are on controls.